src/components/OpenLayersMap.js

```diff
@@ -29,8 +30,41 @@ class OpenLayersMap extends React.Component {
     };
   }
 
+  deleteLastLine = ()=> {
+    const {mapInteractions, map } = this.state;
+    _.each(mapInteractions, i => {
+      if(i.geometryName_ === 'Polygon'){
+        i.removeLastPoint()
+      }
+    })
+  }
+
+  escapeInteractions = ()=> {
+    const {map, mapInteractions } = this.state;
+    _.each(mapInteractions, i => map.removeInteraction(i))
+    this.setState({ mapInteractions: []})
+  }
+
   componentDidMount() {
     this.setupMap();
+
+    const triggerEscape = ()=> {
+      this.escapeInteractions()
+    }
+    const triggerDeleteLast = ()=> {
+      this.deleteLastLine()
+    }
+    document.onkeydown = function(evt) {
+        evt = evt || window.event;
+        if (evt.keyCode == 27) {
+          triggerEscape()
```

Comment:
I'd just have this and the other key event call their actions directly (`this.escapeInteractions()`. The `trigger__` call doesn't add anything, extra complexity.